push_to_docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# The image build step occurs only when a build is triggered by a commit being merged into the "master" branch.
+# Given that the "master" branch is protected, the only way to trigger a build from the "master" branch is when a PR
+# is merged into it. This way, we avoid storing images from non-stable branches.
+VERSION=$(cat ./version/version.go | grep "var VERSION" | awk ' { print $4 } ' | sed s/\"//g)
```

Comment:
Quote version and use single-quotes whenever you don't want expansion to happen:

```bash
VERSION="$(cat ./version/version.go | grep 'var VERSION' | awk '{ print $4 }' | sed 's/"//g')"
```

Btw, I'd much rather replace this with a `go run ./ version` command. I'm not sure why we're using (and trying to parse) a `.go` file that's otherwise unconnected to the program. Either we connect it (by something like a `version` command) or just make it a `.txt` with just the version number, so we remove the need for parsing.

Reply:
I'm not sure if there is a standard way of versioning Go projects in code, so I ended up created the `.go` file based on [this article](https://polyverse.com/blog/how-to-embed-versioning-information-in-go-applications-f76e2579b572/) and [etcd](https://github.com/etcd-io/etcd/blob/master/version/version.go).

Etcd must have its reasons for using a file like that, which are not the same as ours. Gonna make it simple with the `.txt` as you suggested.

Reply:
There are 2 distinct issues with `version.go` as it is (was?):

1. Being disconnected with the program. You cited etcd, and that's a good example, as their versioning info is not disconnected from the program - if you do `etcd --version` it prints out the versioning info. Ours is an unused global variable.

1. The procedure used  to update/release new versions. That doesn't change if we're using `version.go`, `version.txt` or whatever else. I'm not convinced we want to add that process.